### PR TITLE
Rename a8n to campaigns in Go code

### PR DESCRIPTION
**Pull request by mrnugget**
Bye bye a8n/Automation/Automaçion, welcome Campaigns!

### CODE REVIEW

File: cmd/frontend/graphqlbackend/graphqlbackend.go [hunk 1]
```diff
@@ -100,14 +100,14 @@ func (prometheusTracer) TraceField(ctx context.Context, label, typeName, fieldNa
 	}
 }
 
-func NewSchema(a8n A8NResolver, codeIntel CodeIntelResolver, authz AuthzResolver) (*graphql.Schema, error) {
+func NewSchema(campaigns A8NResolver, codeIntel CodeIntelResolver, authz AuthzResolver) (*graphql.Schema, error) {
```
Comment (unknwon) [inline cmd/frontend/graphqlbackend/graphqlbackend.go]:
```suggestion
func NewSchema(campaigns CampaignsResolver, codeIntel CodeIntelResolver, authz AuthzResolver) (*graphql.Schema, error) {
```

File: cmd/frontend/graphqlbackend/graphqlbackend.go [hunk 2]
```diff
@@ -100,14 +100,14 @@ func (prometheusTracer) TraceField(ctx context.Context, label, typeName, fieldNa
...
 	resolver := &schemaResolver{
 		A8NResolver:       defaultA8NResolver{},
```
Comment (unknwon) [inline cmd/frontend/graphqlbackend/graphqlbackend.go]:
```suggestion
		CampaignsResolver:       defaultCampaignsResolver{},
```

Comment (mrnugget) [reply cmd/frontend/graphqlbackend/graphqlbackend.go]: Nice catches! Fixed :)

File: enterprise/internal/campaigns/store.go [hunk 3]
```diff
@@ -324,7 +324,7 @@ WITH batch AS (
 `
 
 var createChangesetsQueryFmtstr = changesetBatchQueryPrefix + `,
--- source: internal/a8n/store.go:CreateChangesets
+-- source: internal/campaigns/store.go:CreateChangesets
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CreateChangesets
```

File: enterprise/internal/campaigns/store.go [hunk 4]
```diff
@@ -454,7 +454,7 @@ func (s *Store) CountChangesets(ctx context.Context, opts CountChangesetsOpts) (
 }
 
 var countChangesetsQueryFmtstr = `
--- source: internal/a8n/store.go:CountChangesets
+-- source: internal/campaigns/store.go:CountChangesets
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CountChangesets
```

File: enterprise/internal/campaigns/store.go [hunk 5]
```diff
@@ -503,7 +503,7 @@ func (s *Store) GetChangeset(ctx context.Context, opts GetChangesetOpts) (*a8n.C
 }
 
 var getChangesetsQueryFmtstr = `
--- source: internal/a8n/store.go:GetChangeset
+-- source: internal/campaigns/store.go:GetChangeset
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetChangeset
```

File: enterprise/internal/campaigns/store.go [hunk 6]
```diff
@@ -573,7 +573,7 @@ func (s *Store) ListChangesets(ctx context.Context, opts ListChangesetsOpts) (cs
 }
 
 var listChangesetsQueryFmtstr = `
--- source: internal/a8n/store.go:ListChangesets
+-- source: internal/campaigns/store.go:ListChangesets
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:ListChangesets
```

File: enterprise/internal/campaigns/store.go [hunk 7]
```diff
@@ -647,7 +647,7 @@ func (s *Store) UpdateChangesets(ctx context.Context, cs ...*a8n.Changeset) erro
 }
 
 const updateChangesetsQueryFmtstr = changesetBatchQueryPrefix + `,
--- source: internal/a8n/store.go:UpdateChangesets
+-- source: internal/campaigns/store.go:UpdateChangesets
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:UpdateChangesets
```

File: enterprise/internal/campaigns/store.go [hunk 8]
```diff
@@ -720,7 +720,7 @@ func (s *Store) GetChangesetEvent(ctx context.Context, opts GetChangesetEventOpt
 }
 
 var getChangesetEventsQueryFmtstr = `
--- source: internal/a8n/store.go:GetChangesetEvent
+-- source: internal/campaigns/store.go:GetChangesetEvent
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetChangesetEvent
```

File: enterprise/internal/campaigns/store.go [hunk 9]
```diff
@@ -786,7 +786,7 @@ func (s *Store) ListChangesetEvents(ctx context.Context, opts ListChangesetEvent
 }
 
 var listChangesetEventsQueryFmtstr = `
--- source: internal/a8n/store.go:ListChangesetEvents
+-- source: internal/campaigns/store.go:ListChangesetEvents
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:ListChangesetEvents
```

File: enterprise/internal/campaigns/store.go [hunk 10]
```diff
@@ -848,7 +848,7 @@ func (s *Store) CountChangesetEvents(ctx context.Context, opts CountChangesetEve
 }
 
 var countChangesetEventsQueryFmtstr = `
--- source: internal/a8n/store.go:CountChangesetEvents
+-- source: internal/campaigns/store.go:CountChangesetEvents
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CountChangesetEvents
```

File: enterprise/internal/campaigns/store.go [hunk 11]
```diff
@@ -918,7 +918,7 @@ ORDER BY batch.ordinality
 `
 
 var upsertChangesetEventsQueryFmtstr = changesetEventsBatchQueryPrefix + `,
--- source: internal/a8n/store.go:UpsertChangesetEvents
+-- source: internal/campaigns/store.go:UpsertChangesetEvents
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:UpsertChangesetEvents
```

File: enterprise/internal/campaigns/store.go [hunk 12]
```diff
@@ -1012,7 +1012,7 @@ func (s *Store) CreateCampaign(ctx context.Context, c *a8n.Campaign) error {
 }
 
 var createCampaignQueryFmtstr = `
--- source: internal/a8n/store.go:CreateCampaign
+-- source: internal/campaigns/store.go:CreateCampaign
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CreateCampaign
```

File: enterprise/internal/campaigns/store.go [hunk 13]
```diff
@@ -1114,7 +1114,7 @@ func (s *Store) UpdateCampaign(ctx context.Context, c *a8n.Campaign) error {
 }
 
 var updateCampaignQueryFmtstr = `
--- source: internal/a8n/store.go:UpdateCampaign
+-- source: internal/campaigns/store.go:UpdateCampaign
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:UpdateCampaign
```

File: enterprise/internal/campaigns/store.go [hunk 14]
```diff
@@ -1180,15 +1180,15 @@ func (s *Store) DeleteCampaign(ctx context.Context, id int64) error {
 }
 
 var deleteCampaignQueryFmtstr = `
--- source: internal/a8n/store.go:DeleteCampaign
+-- source: internal/campaigns/store.go:DeleteCampaign
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:DeleteCampaign
```

File: enterprise/internal/campaigns/store.go [hunk 15]
```diff
@@ -1201,7 +1201,7 @@ func (s *Store) CountCampaigns(ctx context.Context, opts CountCampaignsOpts) (co
 }
 
 var countCampaignsQueryFmtstr = `
--- source: internal/a8n/store.go:CountCampaigns
+-- source: internal/campaigns/store.go:CountCampaigns
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CountCampaigns
```

File: enterprise/internal/campaigns/store.go [hunk 16]
```diff
@@ -1253,7 +1253,7 @@ func (s *Store) GetCampaign(ctx context.Context, opts GetCampaignOpts) (*a8n.Cam
 }
 
 var getCampaignsQueryFmtstr = `
--- source: internal/a8n/store.go:GetCampaign
+-- source: internal/campaigns/store.go:GetCampaign
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetCampaign
```

File: enterprise/internal/campaigns/store.go [hunk 17]
```diff
@@ -1321,7 +1321,7 @@ func (s *Store) ListCampaigns(ctx context.Context, opts ListCampaignsOpts) (cs [
 }
 
 var listCampaignsQueryFmtstr = `
--- source: internal/a8n/store.go:ListCampaigns
+-- source: internal/campaigns/store.go:ListCampaigns
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:ListCampaigns
```

File: enterprise/internal/campaigns/store.go [hunk 18]
```diff
@@ -1383,7 +1383,7 @@ func (s *Store) CreateCampaignPlan(ctx context.Context, c *a8n.CampaignPlan) err
 }
 
 var createCampaignPlanQueryFmtstr = `
--- source: internal/a8n/store.go:CreateCampaignPlan
+-- source: internal/campaigns/store.go:CreateCampaignPlan
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CreateCampaignPlan
```

File: enterprise/internal/campaigns/store.go [hunk 19]
```diff
@@ -1442,7 +1442,7 @@ func (s *Store) UpdateCampaignPlan(ctx context.Context, c *a8n.CampaignPlan) err
 }
 
 var updateCampaignPlanQueryFmtstr = `
--- source: internal/a8n/store.go:UpdateCampaignPlan
+-- source: internal/campaigns/store.go:UpdateCampaignPlan
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:UpdateCampaignPlan
```

File: enterprise/internal/campaigns/store.go [hunk 20]
```diff
@@ -1493,7 +1493,7 @@ func (s *Store) DeleteCampaignPlan(ctx context.Context, id int64) error {
 }
 
 var deleteCampaignPlanQueryFmtstr = `
--- source: internal/a8n/store.go:DeleteCampaignPlan
+-- source: internal/campaigns/store.go:DeleteCampaignPlan
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:DeleteCampaignPlan
```

File: enterprise/internal/campaigns/store.go [hunk 21]
```diff
@@ -1513,7 +1513,7 @@ func (s *Store) DeleteExpiredCampaignPlans(ctx context.Context) error {
 }
 
 var deleteExpiredCampaignPlansQueryFmtstr = `
--- source: internal/a8n/store.go:DeleteExpiredCampaignPlans
+-- source: internal/campaigns/store.go:DeleteExpiredCampaignPlans
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:DeleteExpiredCampaignPlans
```

File: enterprise/internal/campaigns/store.go [hunk 22]
```diff
@@ -1550,7 +1550,7 @@ func (s *Store) CountCampaignPlans(ctx context.Context) (count int64, _ error) {
 }
 
 var countCampaignPlansQueryFmtstr = `
--- source: internal/a8n/store.go:CountCampaignPlans
+-- source: internal/campaigns/store.go:CountCampaignPlans
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CountCampaignPlans
```

File: enterprise/internal/campaigns/store.go [hunk 23]
```diff
@@ -1580,7 +1580,7 @@ func (s *Store) GetCampaignPlan(ctx context.Context, opts GetCampaignPlanOpts) (
 }
 
 var getCampaignPlansQueryFmtstr = `
--- source: internal/a8n/store.go:GetCampaignPlan
+-- source: internal/campaigns/store.go:GetCampaignPlan
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetCampaignPlan
```

File: enterprise/internal/campaigns/store.go [hunk 24]
```diff
@@ -1617,7 +1617,7 @@ func (s *Store) GetCampaignPlanStatus(ctx context.Context, id int64) (*a8n.Backg
 }
 
 var getCampaignPlanStatusQueryFmtstr = `
--- source: internal/a8n/store.go:GetCampaignPlanStatus
+-- source: internal/campaigns/store.go:GetCampaignPlanStatus
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetCampaignPlanStatus
```

File: enterprise/internal/campaigns/store.go [hunk 25]
```diff
@@ -1629,39 +1629,39 @@ WHERE %s
...
 	case status.Completed == status.Total && len(status.ProcessErrors) != 0:
-		status.ProcessState = a8n.BackgroundProcessStateErrored
+		status.ProcessState = campaigns.BackgroundProcessStateErrored
 	}
 	return &status, nil
 }
 
 var getCampaignStatusQueryFmtstr = `
--- source: internal/a8n/store.go:GetCampaignStatus
+-- source: internal/campaigns/store.go:GetCampaignStatus
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetCampaignStatus
```

File: enterprise/internal/campaigns/store.go [hunk 26]
```diff
@@ -1704,7 +1704,7 @@ func (s *Store) ListCampaignPlans(ctx context.Context, opts ListCampaignPlansOpt
 }
 
 var listCampaignPlansQueryFmtstr = `
--- source: internal/a8n/store.go:ListCampaignPlans
+-- source: internal/campaigns/store.go:ListCampaignPlans
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:ListCampaignPlans
```

File: enterprise/internal/campaigns/store.go [hunk 27]
```diff
@@ -1752,7 +1752,7 @@ func (s *Store) CreateCampaignJob(ctx context.Context, c *a8n.CampaignJob) error
 }
 
 var createCampaignJobQueryFmtstr = `
--- source: internal/a8n/store.go:CreateCampaignJob
+-- source: internal/campaigns/store.go:CreateCampaignJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CreateCampaignJob
```

File: enterprise/internal/campaigns/store.go [hunk 28]
```diff
@@ -1821,7 +1821,7 @@ func (s *Store) UpdateCampaignJob(ctx context.Context, c *a8n.CampaignJob) error
 }
 
 var updateCampaignJobQueryFmtstr = `
--- source: internal/a8n/store.go:UpdateCampaignJob
+-- source: internal/campaigns/store.go:UpdateCampaignJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:UpdateCampaignJob
```

File: enterprise/internal/campaigns/store.go [hunk 29]
```diff
@@ -1882,7 +1882,7 @@ func (s *Store) DeleteCampaignJob(ctx context.Context, id int64) error {
 }
 
 var deleteCampaignJobQueryFmtstr = `
--- source: internal/a8n/store.go:DeleteCampaignJob
+-- source: internal/campaigns/store.go:DeleteCampaignJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:DeleteCampaignJob
```

File: enterprise/internal/campaigns/store.go [hunk 30]
```diff
@@ -1909,7 +1909,7 @@ func (s *Store) CountCampaignJobs(ctx context.Context, opts CountCampaignJobsOpt
 }
 
 var countCampaignJobsQueryFmtstr = `
--- source: internal/a8n/store.go:CountCampaignJobs
+-- source: internal/campaigns/store.go:CountCampaignJobs
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CountCampaignJobs
```

File: enterprise/internal/campaigns/store.go [hunk 31]
```diff
@@ -1965,7 +1965,7 @@ func (s *Store) GetCampaignJob(ctx context.Context, opts GetCampaignJobOpts) (*a
 }
 
 var getCampaignJobsQueryFmtstr = `
--- source: internal/a8n/store.go:GetCampaignJob
+-- source: internal/campaigns/store.go:GetCampaignJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetCampaignJob
```

File: enterprise/internal/campaigns/store.go [hunk 32]
```diff
@@ -2035,7 +2035,7 @@ func (s *Store) ListCampaignJobs(ctx context.Context, opts ListCampaignJobsOpts)
 }
 
 var listCampaignJobsQueryFmtstr = `
--- source: internal/a8n/store.go:ListCampaignJobs
+-- source: internal/campaigns/store.go:ListCampaignJobs
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:ListCampaignJobs
```

File: enterprise/internal/campaigns/store.go [hunk 33]
```diff
@@ -2122,7 +2122,7 @@ func (s *Store) CreateChangesetJob(ctx context.Context, c *a8n.ChangesetJob) err
 }
 
 var createChangesetJobQueryFmtstr = `
--- source: internal/a8n/store.go:CreateChangesetJob
+-- source: internal/campaigns/store.go:CreateChangesetJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CreateChangesetJob
```

File: enterprise/internal/campaigns/store.go [hunk 34]
```diff
@@ -2185,7 +2185,7 @@ func (s *Store) UpdateChangesetJob(ctx context.Context, c *a8n.ChangesetJob) err
 }
 
 var updateChangesetJobQueryFmtstr = `
--- source: internal/a8n/store.go:UpdateChangesetJob
+-- source: internal/campaigns/store.go:UpdateChangesetJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:UpdateChangesetJob
```

File: enterprise/internal/campaigns/store.go [hunk 35]
```diff
@@ -2240,7 +2240,7 @@ func (s *Store) DeleteChangesetJob(ctx context.Context, id int64) error {
 }
 
 var deleteChangesetJobQueryFmtstr = `
--- source: internal/a8n/store.go:DeleteChangesetJob
+-- source: internal/campaigns/store.go:DeleteChangesetJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:DeleteChangesetJob
```

File: enterprise/internal/campaigns/store.go [hunk 36]
```diff
@@ -2260,7 +2260,7 @@ func (s *Store) CountChangesetJobs(ctx context.Context, opts CountChangesetJobsO
 }
 
 var countChangesetJobsQueryFmtstr = `
--- source: internal/a8n/store.go:CountChangesetJobs
+-- source: internal/campaigns/store.go:CountChangesetJobs
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:CountChangesetJobs
```

File: enterprise/internal/campaigns/store.go [hunk 37]
```diff
@@ -2335,7 +2335,7 @@ func (s *Store) GetChangesetJob(ctx context.Context, opts GetChangesetJobOpts) (
 }
 
 var getChangesetJobsQueryFmtstr = `
--- source: internal/a8n/store.go:GetChangesetJob
+-- source: internal/campaigns/store.go:GetChangesetJob
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:GetChangesetJob
```

File: enterprise/internal/campaigns/store.go [hunk 38]
```diff
@@ -2409,7 +2409,7 @@ func (s *Store) ListChangesetJobs(ctx context.Context, opts ListChangesetJobsOpt
 }
 
 var listChangesetJobsQueryFmtstrSelect = `
--- source: internal/a8n/store.go:ListChangesetJobs
+-- source: internal/campaigns/store.go:ListChangesetJobs
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:ListChangesetJobs
```

File: enterprise/internal/campaigns/store.go [hunk 39]
```diff
@@ -2498,7 +2498,7 @@ func resetChangesetJobsQuery(campaignID int64, onlyErrored bool) *sqlf.Query {
 }
 
 var resetChangesetJobsQueryFmtstr = `
--- source: internal/a8n/store.go:resetChangesetJobsQuery
+-- source: internal/campaigns/store.go:resetChangesetJobsQuery
```
Comment (unknwon) [inline enterprise/internal/campaigns/store.go]:
```suggestion
-- source: enterprise/internal/campaigns/store.go:resetChangesetJobsQuery
```

**Pull request merged**